CHANGES.md

```diff
@@ -1,6 +1,82 @@
 Release Changes List
 ====================
 
+version 1.22.0
+==============
+
+Twenty-fifth public release of Chapel, April 16, 2020  
+Second release candidate for Chapel 2.0 (RC2)
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* converted major instances of implicit indexing to be 0- rather than 1-based:
+  - tuples
+  - varargs arguments
+  - strings
+  - `bytes`
```

Comment:
I understand why did it this way, but consistency may be better here.

Maybe you can merge them and make it `- string/bytes` so that from the context it is clear when you say "bytes"?

Reply:
See if you like the latest commit better.

Reply:
looks good